lib/rubocop/cop/style/block_delimiters.rb

```diff
@@ -28,7 +28,7 @@ def on_block(node)
           if proper_block_style?(node)
             correct_style_detected
           else
-            add_offense(node, :begin) { opposite_style_detected }
+            add_offense(node, :begin) { unrecognized_style_detected }
```

Comment:
Actually, those `..._style_detected` methods don't do anything useful when there are more than two styles to choose from. So you can remove this block and the call to `correct_style_detected`.